deployer-wp/Dockerfile

```diff
@@ -0,0 +1,71 @@
+FROM php:7.1-cli
+
+LABEL maintainer="Mriyam Tamuli <mbtamuli@gmail.com>, Shantanu Deshpande <shantanud106@gmail.com>"
+
+RUN echo "memory_limit=-1" > "$PHP_INI_DIR/conf.d/memory-limit.ini" \
+&& echo "date.timezone=Asia/Kolkata" > "$PHP_INI_DIR/conf.d/date_timezone.ini"
+
+VOLUME /root/composer
+ENV HOME /root
+ENV PATH "/composer/vendor/bin:~/.local/bin:$PATH"
```

Comment:
If you are using `/composer` as composer home, then what is the use of creating `VOLUME /root/composer`

Reply:
Didn't changed it as it was the part of existing `ci-deployer` image. Just excluded what was added in different image.

Reply:
Can't seem to think of a reason it'll be useful. Please remove it from this image. Later we can also update ci-deployer.

Reply:
Removed from Dockerfile [Remove VOLUME](https://github.com/iamShantanu101/dockerfiles/commit/b3b72fb80ed01b4cf0e50382fbc12397f1856668)